BWT/BWT/BWTAlgorithm.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace BWT
+{
+    using System.Text;
+    public class BWTAlgorithm
+    {
+        public static (string, int) FrontBWT(string input)
+        {
+            int lengthOfString = input.Length;
+            string[] rotations = new string[lengthOfString];
+
+            for (int i = 0; i < lengthOfString; ++i)
+            {
+                rotations[i] = input.Substring(i) + input.Substring(0, i);
```

Comment:
В последнем .net можно вместо Substring использовать оператор диапазона, то есть 
```suggestion
                rotations[i] = input[i..] + input[..i];
```